charts/guac/README.md

```diff
@@ -37,6 +37,10 @@ The full GUAC component deployment is a set of asynchronous services that combin
 - **OSV Certifier**: This certifier gathers OSV vulnerability information from
   [osv.dev](https://osv.dev/) about packages.
 
+- **GUAC Visualizer**: The GUAC Visualizer is an experimental utility that can be used to interact with GUAC services. It acts as a way to visualize the software supply chain graph, as well as a means to explore the supply chain and prototype policies. 
+
+  **Note**: you will need to port-forward to the graphql service for the visualizer to work.
```

Comment:
is this still true? I thought this was only the case when running graphql in cluster and visualizer locally

Reply:
oh right, that would not be needed.